docs/csharp/programming-guide/concepts/linq/join-operations.md

```diff
@@ -21,6 +21,20 @@ A *join* of two data sources is the association of objects in one data source wi
 |Join|Joins two sequences based on key selector functions and extracts pairs of values.|`join … in … on … equals …`|<xref:System.Linq.Enumerable.Join%2A?displayProperty=nameWithType><br /><br /> <xref:System.Linq.Queryable.Join%2A?displayProperty=nameWithType>|  
 |GroupJoin|Joins two sequences based on key selector functions and groups the resulting matches for each element.|`join … in … on … equals … into …`|<xref:System.Linq.Enumerable.GroupJoin%2A?displayProperty=nameWithType><br /><br /> <xref:System.Linq.Queryable.GroupJoin%2A?displayProperty=nameWithType>|  
   
+## Query Expression Syntax Examples  
+  
+### Join  
+  
+The following example uses the `join … in … on … equals …` clause to join two sequences based on specific value:
+  
+[!code-csharp[Join](~/samples/snippets/csharp/VS_Snippets_VBCSharp/CsLINQJoin/CS/JoinOperation.cs#Join)]  
+
+### GroupJoin  
```

Comment:
Same here ?

Reply:
Adding this line below within the YAML header will lock all instances of both "Join" and "GroupJoin":
```
no-loc: [Join, GroupJoin]
```